core/src/raw/ops.rs

```diff
@@ -419,6 +421,28 @@ impl OpRead {
     pub fn executor(&self) -> Option<&Executor> {
         self.executor.as_ref()
     }
+
+    /// Set the If-Modified-Since of the option
+    pub fn with_if_modified_since(mut self, if_modified_since: DateTime<Utc>) -> Self {
+        self.if_modified_since = Some(if_modified_since);
+        self
+    }
+
+    /// Get If-Modified-Since from option
+    pub fn if_modified_since(&self) -> Option<DateTime<Utc>> {
+        self.if_modified_since
+    }
+
+    /// Set the If-Unmodified-Since of the option
+    pub fn with_if_unmodified_since(mut self, if_unmodified_since: DateTime<Utc>) -> Self {
```

Comment:
Using `v: DateTime<Utc>` is clear enough; I don't think the full name is necessary here.